internal/cmd/operator-sdk/generate/packagemanifests/packagemanifests.go

```diff
@@ -35,8 +35,15 @@ towards using bundles by default. Run 'operator-sdk generate bundle -h' for more
 
 Running 'generate packagemanifests' is the first step to publishing your operator to a catalog and/or deploying
```

Comment:
IMO we need first all described that pakgmanifest is an old formated and that now is recommended use the bundle instead.

Reply:
Follow-up

Reply:
This will be handled in https://github.com/operator-framework/enhancements/pull/64 I think.